src/data/finset.lean

```diff
@@ -2048,6 +2089,13 @@ theorem le_max_of_mem {s : finset α} {a b : α} (h₁ : a ∈ s) (h₂ : b ∈
 by rcases @le_sup (with_bot α) _ _ _ _ _ h₁ _ rfl with ⟨b', hb, ab⟩;
    cases h₂.symm.trans hb; assumption
 
+lemma exists_max (s : finset β) (f : β → α) (h : s.nonempty) : ∃ x ∈ s, ∀ x' ∈ s, f x' ≤ f x :=
```

Comment:
`exists_max_image`? Your name does not mention the fact that you are applying a function.

Reply:
This is copied from `exists_min` below

Reply:
Let's change both, per @sgouezel's suggestion.